### PR TITLE
Added exception for Google Two Factor Auth (develop)

### PR DESCRIPTION
**Pull request by gman-php**
The Google Auth module is missing support for Two Factor Auth. If your account has it enabled, you would simply get the standard error which would lead the user thinking his email/password combination was wrong. With this special exception you can catch that and display a proper error message that the user needs to make use of Google App Passwords: https://support.google.com/accounts/answer/185833?hl=en&ctx=ch_b%2F0%2FSmsAuthSettings

### CODE REVIEW

File: pgoapi/auth_google.py [hunk 1]
```diff
@@ -57,6 +57,9 @@ def user_login(self, username, password):
 
         user_login = perform_master_login(username, password, self.GOOGLE_LOGIN_ANDROID_ID, proxy=self._proxy)
 
+        if user_login and 'Error' in user_login and user_login['Error'] == 'NeedsBrowser':
```
Comment (sebastienvercammen) [inline pgoapi/auth_google.py]: Change to:
```
if user_login and user_login.get('Error', None) == 'NeedsBrowser':
```

Comment (gman-php) [inline pgoapi/auth_google.py]: good point! will do later

Comment (nullpixel) [inline pgoapi/auth_google.py]: @gman-php already done!

**Comment by nullpixel** [comment]
Should've rebased this / squashed commits

**Comment by keyphact** [comment]
Yeah, noticed after I clicked merge, I don't have git on mobile but what commits were missed while this branch was stale?

**Comment by nullpixel** [comment]
It wasn't really about stale commits, just the load of merge commits at the top are now in the commit

**Comment by keyphact** [comment]
If there are no stale commits, then it's a minor. While it's not wanted and clutters up history, it isn't the end of the world.

**Comment by nullpixel** [comment]
Yeah, 'tis cool

**Comment by gman-php** [comment]
Thx <3

**Pull request merged**